fuzz/cmp.c

```diff
@@ -176,13 +176,26 @@ int FuzzerTestOneInput(const uint8_t *buf, size_t len)
         return 0;
 
     in = BIO_new(BIO_s_mem());
-    OPENSSL_assert((size_t)BIO_write(in, buf, (int)len) == len);
+    if ((size_t)BIO_write(in, buf, (int)len) != len) {
+        BIO_free(in);
+        OPENSSL_assert(0 && "(size_t)BIO_write(in, buf, (int)len == len");
```

Comment:
Please drop this assert and just return 0. This is a regular error condition IMO.

Reply:
Thank you. I have submitted the change.